uxdot/uxdot-color-scheme-picker.css

```diff
@@ -0,0 +1,82 @@
+:host {
+  display: contents;
+}
+
+fieldset {
+  border: 0;
+  margin: 0;
+  padding: 0;
+  display: inline-flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  gap: var(--rh-space-lg);
+
+  legend {
+    float: inline-start;
+  }
+
+  #button-group {
+    display: flex;
+
+    --rh-icon-size: var(--rh-size-icon-01);
+  }
+
+  label {
+    position: relative;
+    background-color: none;
```

Comment:
Not a valid property

<img width="205" alt="image" src="https://github.com/user-attachments/assets/52034414-8aec-43e7-a11c-e952e3fac3c6" />

Reply:
`transparent` works

Reply:
thanks, fixed